code/modules/reagents/chemistry/machinery/chem_master.dm

```diff
@@ -472,10 +518,46 @@
 	desc = "Used to create condiments and other cooking supplies."
 	condi = TRUE
 
+/obj/machinery/chem_master/primitive
+	name = "alchemy table"
+	desc = "A wooden table with various bone mortars and pistles, as well as other tools."
+	icon_state = "alchemy_table"
+	primitive = TRUE
+	use_power = FALSE
+	idle_power_usage = 0
+	flags_1 = NODECONSTRUCT_1
+	can_be_unanchored = TRUE
+
+/obj/machinery/chem_master/primitive/update_icon_state()
+	if(beaker)
+		icon_state = "alchemy_table"
+	else
+		icon_state = "alchemy_table"
+
+/obj/machinery/chem_master/primitive/attackby(obj/item/I, mob/user, params)
+		return
+
+/obj/machinery/chem_master/primitive/ui_interact(mob/living/user, datum/tgui/ui)
+	ui = SStgui.try_update_ui(user, src, ui)
+	if(istype(user, /mob/dead/observer))
+		if(!ui)
+			ui = new(user, src, "ChemMaster", name)
+			ui.open()
+	else
+		if(!HAS_TRAIT(user, TRAIT_MACHINE_SPIRITS) && !istype(src, /obj/machinery/chem_dispenser/drinks))
```

Comment:
`src` will never be (a subtype of) `/obj/machinery/chem_dispenser/drinks`, because `src` is `/obj/machinery/chem_master/primitive`. Are you your cousin? No. Will you ever be your cousin? No.

Reply:
Oversight, changed to:

```
if(!HAS_TRAIT(user, TRAIT_MACHINE_SPIRITS) && !istype(src, /obj/machinery/chem_master/condimaster))
	to_chat(user, "<span class='warning'>Try as you might, you have no clue how to work this thing.</span>")
	return
```